src/collection/components/CollectionOrBundleEdit.tsx

```diff
@@ -1193,25 +1208,27 @@ const CollectionOrBundleEdit: FunctionComponent<
 					<HeaderButtons>
 						{isMobileWidth() ? renderHeaderButtonsMobile() : renderHeaderButtons()}
 					</HeaderButtons>
+
 					<HeaderAvatar>{profile && renderAvatar(profile, { dark: true })}</HeaderAvatar>
 				</Header>
+
 				<Navbar background="alt" placement="top" autoHeight>
 					<Container mode="horizontal">
 						<Tabs tabs={tabs} onClick={selectTab} />
 					</Container>
 				</Navbar>
-				{renderTab()}
-				<Container background="alt" mode="vertical">
-					<Container mode="horizontal">
-						<Toolbar autoHeight>
-							<ToolbarLeft>
-								<ToolbarItem>
-									<ButtonToolbar>{renderSaveButton()}</ButtonToolbar>
```

Comment:
shouldn't the other save button also be removed?
![image](https://user-images.githubusercontent.com/1710840/193843855-403659c1-6971-4d98-aec1-ed99d7ba6211.png)

Reply:
Not in my opinion, ticket only mentions adding the blue bar and doesn't even mention removing the save button at the bottom. It's also useful for existing users, especially here, to keep the button at the top as they've been using it for years.

Reply:
https://studiohyperdrive.slack.com/archives/CJ55B3Q2W/p1664991960911719?thread_ts=1664978539.792149&cid=CJ55B3Q2W